setup.py

```diff
@@ -350,73 +419,54 @@ def run(self):
         self._compile_webhost(self.webhost_dir)
 
 
-with open("README.md") as readme:
-    long_description = readme.read()
+class Clean(distutils.cmd.Command):
+    description = 'Clean up build generated files'
+    user_options = []
```

Comment:
I don't see this used anywhere. Can be removed?

Reply:
Same as above - for implementing the abstract methods and user_options are for it.